api/app/src/main/kotlin/packit/repository/UserRepository.kt

```diff
@@ -0,0 +1,15 @@
+package packit.repository
+
+import org.springframework.data.jpa.repository.JpaRepository
+import org.springframework.stereotype.Repository
+import packit.model.User
+import java.util.*
+
+@Repository
+interface UserRepository : JpaRepository<User, UUID>
+{
+    fun findByUsername(username: String): User?
+    fun findByEmail(email: String): User?
+    fun existsByUsername(username: String): Boolean
```

Comment:
I think it should just be findByUsername and existsByUsername that you need..

Reply:
as per comment below will need both if we keep email